packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -119,7 +119,6 @@ function NetworkListbox({
       onChange={onChange}
     >
       <Listbox.Button
-        disabled={disabled}
```

Comment:
Unnecessary and eslint complaining about it not being valid. Tested and still works (parent element already has `disabled` on it)